Setup/UpgradeSchema.php

```diff
@@ -150,5 +150,27 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
 
             $installer->endSetup();
         }
+
+        if (version_compare($context->getVersion(), '1.0.6', '<')) {
+            $installer = $setup;
+            $installer->startSetup();
+
+            /**
+             * Update table 'sales_order_item'
+             */
+            $installer->getConnection()->addColumn(
+                $installer->getTable('sales_order_item'),
+                'tj_ptc',
+                [
+                    'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                    'length' => 255,
```

Comment:
We don't have any PTCs as long as 255 characters, do we? I'm not familiar with the database behind this, but do we want to size this column to fit? We could still give ourselves a little wiggle room if we would like, but 255 seems really high.

Reply:
You're right; we use 32 elsewhere.  I'll update it!

Reply:
![Screen Shot 2020-10-28 at 6 05 49 PM](https://user-images.githubusercontent.com/44789510/97509569-3efc6480-1948-11eb-882f-849946f452dc.png)